authentication-guides/jira/README.md

```diff
@@ -0,0 +1,267 @@
+                        "jira": "2024-01-10T00:51:48.461+0000",
+                        "friendly": "Today 12:51 AM",
+                        "epochMillis": 1704847908461
+                    },
+                    "_links": {
+                        "self": "https://moveworksus.atlassian.net/rest/servicedeskapi/request/161173/approval/60"
+                    }
+                }
+            ]
+    ```
+
+3. Now that we have access to all approvers and can loop through them, we can add the final pieces for validation and leverage the Event Webhook we created earlier.  As a validation precaution, add a new component “IF: Add a Condition”.  Then choose {{smart values}} condition.  We can now reference fields within the variable we captured above so let’s check and make the sure the emailAddress is “not null” before sending the message to the user.
+
+    ![Screenshot 2024-01-09 at 6.29.25 PM.png](Use%20Case%20Guide%20JIRA%20Approvals%20f1ce3fe611e644a99a97f00cfb719fbb/Screenshot_2024-01-09_at_6.29.25_PM.png)
+
+    ![Screenshot 2024-01-09 at 6.29.48 PM.png](Use%20Case%20Guide%20JIRA%20Approvals%20f1ce3fe611e644a99a97f00cfb719fbb/Screenshot_2024-01-09_at_6.29.48_PM.png)
+
+4. Now for the final step we will send the approval message to the user.  Select new component and “Send Web Request”.  For the Web Request URL enter where EventID is the value you copied when you created the event in Creator Studio
+
+```text
+https://api.moveworks.ai/rest/v1/events/{{EventID}}/messages/send
+```
+
+1. Your HTTP method will be POST and your Web Request Body will be “Custom Data”
+
+    ![Screenshot 2024-01-09 at 6.33.17 PM.png](Use%20Case%20Guide%20JIRA%20Approvals%20f1ce3fe611e644a99a97f00cfb719fbb/Screenshot_2024-01-09_at_6.33.17_PM.png)
+
+2. Within the custom data body we will formulate the message you are going to send to the user.  One thing to note - you can reference ANY field in the issue that came back from the JSON response in the Postman call.  You’ll notice I’m referencing the creator’s email address {{creator.emailAddress}} and the issue type {{issueType.name}} so I can show the approver this info, but it is not limited to this.  See the code snipped below - we are also referencing the {{issueKey}} which allows us to dynamically build the link to the issue, as well as the {{approvers.approver.emailAddress}} so Moveworks knows who to send this message to as the approver
+
+```json
+{
+    "message": "You have a new pending approval in JIRA.  It has been requested by {{creator.emailAddress}} and is of type \"{{issueType.name}}\".  Please view it <a href=\"https://moveworksus.atlassian.net/jira/servicedesk/projects/LEG/queues/custom/58/{{issue.key}}\">here</a>",
+    "recipients": ["{{approvers.approver.emailAddress}}"]
+}
+```
+
+1. Once this is done, the final piece is the Authorization.  Your customer success team will provide you this bearer token and you plug it in here like so.
```

Comment:
Again remove reference to CS team. Key management is done here. 
https://developer.moveworks.com/creator-studio/integrations/inbound/credentials-management/